capplets/time-admin/src/time-admin.ui

```diff
@@ -0,0 +1,375 @@
+                        <property name="can_focus">True</property>
+                        <property name="max_length">2</property>
+                        <property name="caps_lock_warning">False</property>
+                        <property name="progress_pulse_step">1</property>
+                        <property name="input_purpose">digits</property>
+                        <property name="adjustment">adjustment3</property>
+                        <signal name="changed" handler="on_spin3_changed" swapped="no"/>
+                      </object>
+                      <packing>
+                        <property name="expand">False</property>
+                        <property name="fill">True</property>
+                        <property name="position">2</property>
+                      </packing>
+                    </child>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child type="label">
+              <object class="GtkLabel">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="label" translatable="yes">Time</property>
+                <attributes>
+                  <attribute name="weight" value="bold"/>
+                </attributes>
+              </object>
+            </child>
+          </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+            <property name="position">1</property>
+          </packing>
+        </child>
+        <child>
+          <object class="GtkFrame">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="margin_left">12</property>
+            <property name="margin_right">12</property>
+            <property name="margin_top">6</property>
+            <property name="label_xalign">0</property>
+            <property name="shadow_type">none</property>
+            <child>
+              <object class="GtkAlignment">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="margin_left">6</property>
+                <property name="margin_right">6</property>
+                <property name="margin_top">6</property>
+                <property name="left_padding">12</property>
+                <child>
+                  <object class="GtkCalendar" id="calendar1">
+                    <property name="visible">True</property>
+                    <property name="sensitive" bind-source="button2" bind-property="sensitive"/>
+                    <property name="can_focus">True</property>
+                    <property name="year">2019</property>
+                    <property name="month">5</property>
+                    <property name="day">20</property>
```

Comment:
If I disable NTP, next time I open `mate-time-admin` I get this date - it doesn't auto-update nor responds to Save

Reply:
Works perfectly now!